jwst/tweakreg/tweakreg_catalog.py

```diff
@@ -68,7 +68,9 @@ def make_tweakreg_catalog(model, kernel_fwhm, snr_threshold, sharplo=0.2,
         raise TypeError('The input model must be an ImageModel.')
 
     # Mask the non-imaging area (e.g. MIRI)
-    coverage_mask = (dqflags.pixel['NON_SCIENCE'] & model.dq).astype(bool)
+    coverage_mask = ((dqflags.pixel['NON_SCIENCE'] +
+                      dqflags.pixel['DO_NOT_USE']) &
```

Comment:
Not real good with python boolean logic, but will this give True results for pixels that have either NON_SCIENCE or DO_NOT_USE set, or only both at the same time?  We want the former (i.e. an OR operation, as opposed to AND).

Reply:
It will give OR. Here is an illustration:
```python
In [1]: non_sci = 1

In [2]: badpix = 2

In [3]: dq = 2 + 8 + 16

In [4]: non_sci & dq
Out[4]: 0

In [5]: badpix & dq
Out[5]: 2

In [6]: (badpix + non_sci) & dq
Out[6]: 2

Reply:
Cool. Thanks!

Reply:
To better visualize this, it would help to use the `bin()` function and keep in ming that `&` is a bitwise operation (compares bits in the same positions):
```python
In [8]: mask = badpix + non_sci

In [9]: bin(mask)
Out[9]: '0b11'

In [10]: bin(dq)
Out[10]: '0b11010'

In [11]: bin(mask & dq)
Out[11]: '0b10'
```
So: mask is `00011` while DQ flag is `11010` and doing & on each pair of bits at the same location in both variables, we get:
```
 00011
&
 11010
 -----
 00010
```